### PR TITLE
models refactor

### PR DESCRIPTION
**Pull request by aarek**

### CODE REVIEW

File: app/models/answer.rb [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
-    negative: 3
-  }
   DEFAULT_VALUES = {
     positive: 'Yes',
     neutral: 'Undecided',
     negative: 'No'
   }
 
-  belongs_to :debate
+  enum answer_type: { positive: 1, neutral: 2, negative: 3 }
```
Comment (jestemkarol) [inline app/models/answer.rb]: You can simplify it to: `enum answer_type: %i(positive neutral negative)`

File: spec/models/answer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    context 'negative answer' do
+      it 'has value "No"' do
+        negative_answer = default_answers.find(&answer_by_type(:negative))
+        expect(negative_answer.value).to eq('No')
+      end
+    end
+  end
+
+
+  describe 'class value methods' do
```
Comment (jestemkarol) [inline spec/models/answer_spec.rb:65]: I would rename it just to '#value'

File: spec/models/auth_token_spec.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe AuthToken, type: :model do
+  it 'generates token value' do
+    auth_token = create(:auth_token)
+    expect(auth_token.value).not_to be_blank
+  end
+
+  it 'should validate debate_id' do
```
Comment (jestemkarol) [inline spec/models/auth_token_spec.rb]: Do not use `should`, use `validates debate_id` instead

File: spec/models/answer_spec.rb (hunk 2 above)
Comment (aarek) [reply spec/models/answer_spec.rb:65]: I could but wouldn't it suggest existence of `#value` method?

File: app/models/answer.rb (hunk 1 above)
Comment (aarek) [reply app/models/answer.rb]: I left it this way for backwards compatibility for now.

When enum is set with array of symbols then values start from `0`:

``` ruby
Answer.answer_types
#=> {"positive"=>0, "neutral"=>1, "negative"=>2}
```

will check if nothing uses this values directly and change if it is possible.

File: spec/models/answer_spec.rb (hunk 2 above)
Comment (jestemkarol) [reply spec/models/answer_spec.rb:65]: You're right. Leave it how it is.

**Pull request merged**